tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -2402,6 +2431,33 @@ public void handleEnd() {
                 }
             }
         });
+
+        fixList.add('V', "Fix values that would inherit laterally", new CLDRFilter() {
+            boolean skip;
+            @Override
+            public void handleStart() {
+                // skip if the locale id's parent isn't root. That is, it must be at level-1 locale.
+                skip = !XMLSource.ROOT_ID.equals(LocaleIDParser.getParent(getLocaleID()));
+            }
+            @Override
+            public void handlePath(String xpath) {
+                if (skip) {
+                    return;
+                }
+                String value = cldrFileToFilter.getStringValue(xpath);
+                if (CldrUtility.INHERITANCE_MARKER.equals(value)) {
+                    Output<String> pathWhereFound = new Output<>();
+                    Output<String> localeWhereFound = new Output<>();
+                    String baileyValue = getResolved().getBaileyValue(xpath, pathWhereFound, localeWhereFound);
+                    if (baileyValue != null
+                        && !xpath.equals(pathWhereFound.value)
+                        && !"constructed".equals(pathWhereFound.value)) {
```

Comment:
instead of hard-coded "constructed" this really should be GlossonymConstructor.PSEUDO_PATH

GlossonymConstructor.java:
    public static final String PSEUDO_PATH = "constructed";

Reply:
I think one problem we have is that these magic constants are scattered around, making it difficult to find them (or know that they are there). And that includes utilities like LocaleIDParser.getParent (I always have to search for the one class that has the right method. CLDRUtility, no, LanguageTagParser, no, CLDRLocale, no ...). We might want to have a techdebt cleanup for those little annoyances.

Reply:
It's definitely an issue.  I guess my point is that the clients who will encounter the constant are working at the level of CLDRLocale.getBaileyValue() so a 'higher level' location for the constant (CLDRUtility would make sense).  Most clients who encounter this value aren't going to call GlossonymConstructor directly.

Reply:
All of the XPath related constants should go in one place